curp/src/server/raw_curp/mod.rs

```diff
@@ -321,13 +323,17 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
 
     /// Handle `publish` request
     pub(super) fn handle_publish(&self, req: PublishRequest) -> Result<(), CurpError> {
```

Comment:
We can pass `propose_id` directly to this method, which is consistent with the other methods

Reply:
We need convert `PublishRequest` into `EntryData` while inserting into logs.